release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -424,6 +424,11 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-19-installation-and-update-remove-terraform-ibm-cloud_{context}"]
+==== Cluster API replaces Terraform for {ibm-cloud-title} installations
+
+In {product-title} {ocp-nplus1}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```

Comment:
It seems, from a quick search in Atom, that _the Cluster API_ is far more common. 

```suggestion
In {product-title} {ocp-nplus1}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```

Reply:
I see `{ocp-nplus1}` used in the docs only once. It is [intended to be used to specify the next OCP version](https://github.com/openshift/openshift-docs/pull/86479), not the current. And, I think it will get updated each version. 
```suggestion
In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```

Reply:
It seems, from a quick search in Atom, that _the Cluster API_ is far more common. 

```suggestion
In {product-title} {ocp-nplus1}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
```